web-exercise/ex-4/script.js

```diff
@@ -0,0 +1,23 @@
+const user = {
+    name: "Piyush Sharma",
+    designation: "Senior Software Engineer",
+    company: "Infuse Consulting",
+    hobbies: ["Reading", "Listening to music", "Collecting stamps"]
+}
+
+/**
+ * Use destructuting to log the following
+*/
+
+const printUserProfile = () => {
+    // Piyush Sharma is a Senior Software Engineer at Infuse Consulting. He likes Reading, Listening to music and Collecting stamps
+    var {name, designation, company, hobbies} = user;
+    var [hobby1,hobby2,hobby3]=hobbies
```

Comment:
Avoid using `var` in JavaScript, use `const` and `let` in general and here `const`. Read [JavaScript: Var, Let, or Const? Which One Should you Use?](https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa)